### ISSUE TITLE
Custom icons size in group editor is not standardized 

### ISSUE THREAD

**Issue by IkutiDev** [godotengine/godot#92361]
### Tested versions

- Reproducible in v4.2.2.stable.official [15073afe3]

### System information

Windows 11 Godot v4.2.2.stable.official [15073afe3]

### Issue description

When assigning nodes inside group editor the sizes of custom icons are equal to the original picture size instead of being standardized to match other editor icons. Example screenshot:
![obraz](https://github.com/godotengine/godot/assets/40697587/e64e4d6f-3ea3-4033-9810-088759e1676e)

 Edit: Important to add I saw similar issues with other parts of editor which has been fixed since, so I'm guessing group editor just wasn't caught before, but it might not be too difficult to use the code that was used for inspector part of UI before.

### Steps to reproduce

1. Create a scene with some nodes that have custom editor icons that are normally bigger than standard editor icons
2. Go to Node tab and select Groups category
3. Press Manage groups
4. Write a group name and press add
5. See that both list nodes not in group and nodes in group show the original size of custom icons which can be large instead of being similar to standard editor icon sizes, making using group editor difficult

### Minimal reproduction project (MRP)

N/A

**Comment by mews6** [comment]
could i try to check this out?

**Comment by mews6** [comment]
Little update: it seems to be replicable in a linux build of v4.2.2.stable.official [https://github.com/godotengine/godot/commit/15073afe3856abd2aa1622492fe50026c7d63dc1]() (building from Pop!_OS 22.04). I'm new to the project so i might have to dig around for a while to check what is causing it. Maybe it isn't resizing the icons properly or something, but it doesn't seem OS-dependant. If someone has different results, please let me know.

**Comment by IkutiDev** [comment]
@mews6 I don't believe it's os oriented, my current guess is that it's related to this old issue https://github.com/godotengine/godot/issues/68962 where it was bugged for inspector and just that fixed wasn't added to group editor UI (guessing it's less used, most people assign groups via code and possibly don't use custom icons for nodes).

No one seems to be taking it right now so you are free to poke around. I would personally look into the PR that fixed the issue inside inspector. And than check if that fix can be applied to the file that creates the group editor UI.

**Comment by cselip** [comment]
@IkutiDev would this just be groups_editor.cpp?

**Comment by IkutiDev** [comment]
@cselip Not 100% sure myself but I believe so, from a quick glance.

**Comment by mews6** [comment]
Currently taking a look at the file @cselip thinks might be the root of the problem, trying to check for anywhere where icons might be rezised, or if such a function even exists.

**Comment by Dowsley** [comment]
@mews6 this is the commit that adds resizing to the Editor https://github.com/godotengine/godot/pull/75472/commits/1522762dc986c130ad63cbf854514d730788a4cf

It seems to employ different strategies for each part of the editor.

**Comment by Dowsley** [comment]
https://github.com/godotengine/godot/pull/60965 removed the Group Editor and replaced it with an alternative that does not show which nodes that belong/don't belong in each group.

Thus solving the problem. For now... 😊

**Comment by elchukc** [comment]
> #60965 removed the Group Editor and replaced it with an alternative that does not show which nodes that belong/don't belong in each group.
> 
> No node icon = problem solved. At least for now 😊

Do we close this issue then?

**Comment by Honey-Sekhon** [comment]
Hello I tried to reproduce this bug in 4.4.dev version after following the instructions described in the issue, but i was unable to do so.I have tried several different images but the custom icons are automatically resized.
Do I need to specifically test it on v4.2.2.stable.official

**Comment by Calinou** [comment]
This was resolved by https://github.com/godotengine/godot/pull/60965, closing.
